lib/kafka/schema_registry/service.rb

```diff
@@ -47,49 +48,40 @@ def check(subject, schema)
       # - nil if the subject or version does not exist
       # - false if incompatible
       # http://docs.confluent.io/3.1.2/schema-registry/docs/api.html#compatibility
-      def compatible?(subject, schema, version = 'latest')
-        data = post("/compatibility/subjects/#{subject}/versions/#{version}",
-                    expects: [200, 404], body: { schema: schema.to_s }.to_json, idempotent: true)
-        data.fetch('is_compatible', false) unless data.key?('error_code')
+      def compatible?(topic, schema, version = 'latest')
+        data = request("/compatibility/subjects/#{topic}-value/versions/#{version}",
+                       method: :post,
+                       expects: [200, 404],
+                       body: { schema: schema.to_s }.to_json,
+                       idempotent: true)
+        data.fetch(:is_compatible, false) unless data.key?('error_code')
```

Comment:
not sure why but the response gets change from a string key to a symbol. not sure why that would be different from the avro_turf gem but that's what i had to do to get specs working.